Controllers/controller.py

```diff
@@ -349,3 +358,20 @@ def toggle_play_pause_icon(self):
             button.setIcon(button.style().standardIcon(QStyle.SP_MediaPause))
         else:
             button.setIcon(button.style().standardIcon(QStyle.SP_MediaPlay))
+
```

Comment:
Make sure you add comments to the following functions. Based on the format we're using right now.